cookbooks/bcpc-hadoop/recipes/datanode.rb

```diff
@@ -18,6 +18,21 @@
   end
 end
 
+user_ulimit "hdfs" do
```

Comment:
Should [hadoop-hdfs-datanode service](https://github.com/cbaenziger/chef-bach/blob/ulimit/cookbooks/bcpc-hadoop/recipes/datanode.rb#L236-L240) subscribe to this resource?